### ISSUE TITLE
Manifest does not contain icon image

### ISSUE THREAD

**Issue by exoRift** [Submitty/Submitty#8616]
**Describe the bug**
You have a manifest which allows "installation" of the website on Android, but the manifest doesn't have the icon which results in a default icon

**Expected behavior**
The rubber duck icon should be there

**To Reproduce**
1. Open Chrome on Android
2. Dropdown the three-dot menu
3. Tap "Install Submitty"

**Configuration**
 - OS: Android
 - Browser: Chrome

**Screenshots**
![img](https://i.imgur.com/PeBF0qH.jpg)

**Comment by 852hamza** [comment]
can you assign me

**Comment by ziesski** [comment]
I was able to reproduce the issue, will work on it.

**Comment by exoRift** [comment]
This issue seems to be persisting even after this fix.

For more information, I'm running Android 13 on a Samsung Galaxy S20+ and using Action Launcher

**Comment by ziesski** [comment]
Thanks for your attention, I will check if I can reproduce the issue, and fix the issue :)

**Comment by exoRift** [comment]
> > This issue seems to be persisting even after this fix.
> > For more information, I'm running Android 13 on a Samsung Galaxy S20+ and using Action Launcher
> > I've tried clearing Chrome cache, site cache, and restarting my phone.
> 
> Hi @exoRift ,
> 
> I was not able to reproduce the issue with Samsung Galaxy Note 20 / Android 13. �Android has two main types of screens: Home screen and App drawer. Does the default icon show up for both screens? I am guessing that it may have to do with Action Launcher, which is third party app. Could you try using default launcher and check if the issue persists?
> 
> Thanks

Yeah, I've tried it with One UI launcher (stock) as well. Did not fix it. The issue persists in the app drawer as well as the app page in the settings

![image](https://github.com/Submitty/Submitty/assets/20345400/0d599b85-2454-481d-8a56-7d80f23ffc6a)
